File: Python/BEGINNER/1064 Positives and Average.py
n1 = float(input())
n2 = float(input())
n3 = float(input())
n4 = float(input())
n5 = float(input())
n6 = float(input())
sum = cont = 0

if n1 > 0:
    cont += 1
    sum += n1
if n2 > 0:
    cont += 1
    sum += n2
if n3 > 0:
    cont += 1
    sum += n3
if n4 > 0:
    cont += 1
    sum += n4
if n5 > 0:
    cont += 1
    sum += n5
if n6 > 0:
    cont += 1
    sum += n6

print("%d valores positivos" % int(cont))
print("%.1f" % (sum/cont))

File: C/BEGINNER/1064 Positives and Average.c
#include <stdio.h>

int main()
{
    double v1, v2, v3, v4, v5, v6, average, sumPositives = 0;
    int numPositives = 0;
    scanf("%lf", &v1);
    scanf("%lf", &v2);
    scanf("%lf", &v3);
    scanf("%lf", &v4);
    scanf("%lf", &v5);
    scanf("%lf", &v6);

    if (v1 > 0)
    {
        sumPositives += v1;
        numPositives += 1;
    }
    if (v2 > 0)
    {
        sumPositives += v2;
        numPositives += 1;
    }
    if (v3 > 0)
    {
        sumPositives += v3;
        numPositives += 1;
    }
    if (v4 > 0)
    {
        sumPositives += v4;
        numPositives += 1;
    }
    if (v5 > 0)
    {
        sumPositives += v5;
        numPositives += 1;
    }
    if (v6 > 0)
    {
        sumPositives += v6;
        numPositives += 1;
    }
    
    average = (sumPositives / numPositives);

    printf("%d valores positivos\n", numPositives);
    printf("%.1lf\n", average);
    return 0;
}
